apple/Handlers/RNPanHandler.h

```diff
@@ -9,4 +9,7 @@
 #import "RNGestureHandler.h"
 
 @interface RNPanGestureHandler : RNGestureHandler
+#if !TARGET_OS_OSX && !TARGET_OS_TV
+@property (atomic, assign) StylusData *stylusData;
```

Comment:
I'm not a fan of keeping reference to the same StylusData both from the recognizer and the handler, especially when they both have access to each other.

Reply:
Totally agree with you! Changed in [e120bac](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/e120bacfea13e2af18a0757a977715ce22358a50)[e120bac](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/e120bacfea13e2af18a0757a977715ce22358a50)